zproject/urls.py

```diff
@@ -264,6 +264,8 @@
     # (this API is only used externally)
     url(r'^streams$', rest_dispatch,
         {'GET': 'zerver.views.streams.get_streams_backend'}),
+    url(r'^streams/(?P<stream_name>.*)$', rest_dispatch,
+        {'PATCH': 'zerver.views.streams.update_stream_backend'}),
```

Comment:
I guess you can remove this because you already have the same route at:
https://github.com/anirudhjain75/zulip/blob/548af190bd517100a52472484550a73bab8946c5/zproject/urls.py#L276